src/components/Accounts/ChargeFeeFine/ChargeFeeFine.js

```diff
@@ -257,9 +257,9 @@ class ChargeFeeFine extends React.Component {
     this.setState({ ownerId });
   }
 
-  onChangeFeeFine(e) {
+  onChangeFeeFine(feeFineId) {
```

Comment:
@ymacedor the `onChagenFeeFine` is attached to `onChange` event:

https://github.com/folio-org/ui-users/blob/d48a3bc02fd5a5e32395544fe0bc801c890a973a/src/components/Accounts/ChargeFeeFine/FeeFineInfo.js#L97

which passes an event object. Is there a reason you changed it here to `feeFineId`?